src/scheduler.rs

```diff
@@ -514,8 +514,18 @@ async fn inbox_fetch_idle(ctx: &Context, imap: &mut Imap, mut session: Session)
                     warn!(ctx, "Can't set Config::FetchedExistingMsgs: {:#}", err);
                 }
 
-                if let Err(err) = imap.fetch_existing_msgs(ctx, &mut session).await {
-                    warn!(ctx, "Failed to fetch existing messages: {:#}", err);
+                match imap.fetch_existing_msgs(ctx, &mut session).await {
+                    Err(err) => {
+                        warn!(ctx, "Failed to fetch existing messages: {:#}", err);
+                    }
+                    Ok(count) => {
+                        let mut msg = Message::new_text(format!(
+                            "Added {} contacts from outgoing chats",
```

Comment:
```suggestion
                            "Added {} contacts from outgoing messages.",
```
It seems we don't have the term "outgoing chat", at least i don't see it in the core code / Delta Chat FAQ.